src/pip/_internal/req/__init__.py

```diff
@@ -46,7 +47,7 @@ def install_given_reqs(
 
     (to be called after having downloaded and unpacked the packages)
     """
-
+    logger.info(get_pip_version())
```

Comment:
As noted in another comment by @pradyunsg , a better place for this would be somewhere in [commands.install:run](https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/install.py#L238) to show the message at the start , but you might have to experiment a bit about where to put the `logger.debug` message within that function.

Reply:
I also saw that there was a request about adding the install location of the packages too, which I think is currently missing. 

In that affect, having a custom function which gets this information along with pip version, and then writing a custom logger message might be better than directly relying on `get_pip_version`.

Reply:
The reason on putting the logger call here was mainly due to this comment https://github.com/pypa/pip/issues/3166#issuecomment-630223130

Yeah, though I guess, I would look for other possible places as well.